src/Http/Routing/src/Builder/FallbackEndpointRouteBuilderExtensions.cs

```diff
@@ -36,7 +36,6 @@ public static class FallbackEndpointRouteBuilderExtensions
     /// <c>{*path:nonfile}</c>. The order of the registered endpoint will be <c>int.MaxValue</c>.
     /// </para>
     /// </remarks>
-    [RequiresUnreferencedCode(EndpointRouteBuilderExtensions.MapEndpointTrimmerWarning)]
```

Comment:
These were just wrong. We could backport removing this to .NET 7 if anyone raises them as an issue.